assembly/std/index.ts

```diff
@@ -37,15 +87,21 @@ export function call(
 }
 
 /**
- * Calls a remote function located at given address within the current context.
+ * Calls a function from a remote contract in the current context.
```

Comment:
Does it mean that the remote function is executed in the local context  ?

Reply:
Yes, you want me to add 'current local context' instead ?